internal/manifests/collector/volume.go

```diff
@@ -42,5 +42,20 @@ func Volumes(cfg config.Config, otelcol v1alpha1.OpenTelemetryCollector) []corev
 		volumes = append(volumes, otelcol.Spec.Volumes...)
 	}
 
+	if len(otelcol.Spec.ConfigMaps) > 0 {
+		for _, cfgMap := range otelcol.Spec.ConfigMaps {
```

Comment:
I think we also need to update the container mounts as well. Otherwise users will need to still manually specify what they want mounted.

Reply:
[ex](https://github.com/prometheus-operator/prometheus-operator/blob/9cc6a9dbde4b7c1e8a24df651f36e81a99c342ab/pkg/prometheus/statefulset.go#L349-L363)